<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WAFFLE-Copy-Professional</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>WaffleUI/Amiga.ico</normaloff>WaffleUI/Amiga.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="mainWindow">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>594</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>594</height>
    </size>
   </property>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>594</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>594</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>594</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WaffleUI/main.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="rightLogo">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>20</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WaffleUI/rightLogo.png</pixmap>
    </property>
   </widget>
   <widget class="QSpinBox" name="serial">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>45</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="ClickLabel" name="writeVerify">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WaffleUI/nVerifyWrite.png</pixmap>
    </property>
   </widget>
   <widget class="ClickLabel" name="startWrite">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WaffleUI/nStart.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="leftLogo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WaffleUI/leftLogo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="portSel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WaffleUI/SerialPort.png</pixmap>
    </property>
   </widget>
   <widget class="ClickLabel" name="runDiagnostic">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WaffleUI/RunDiagnostic.png</pixmap>
    </property>
   </widget>
   <widget class="ClickLabel" name="startRead">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WaffleUI/nStart.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="mainTitle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>681</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WaffleUI/waffleCopyPRO.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="setADFFileName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>select file name to read from Waffle</string>
    </property>
   </widget>
   <widget class="QToolButton" name="fileSaveADF">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>29</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="getADFFileName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>select ADF to write to Waffle</string>
    </property>
   </widget>
   <widget class="QToolButton" name="fileReadADF">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>29</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>clicklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
